lib/src/layer.dart

```diff
@@ -49,6 +51,10 @@ class Layer {
 
       assembleTileMatrix(inflatedString);
     }
+
+    properties = TileMapParser._parseProperties(
+      TileMapParser._getPropertyNodes(element),
+    );
```

Comment:
it seems it would be useful to have a TileMapParser._parsePropertiesFromXML method to be used in 3 places